dss_datamover/config/config.json

```diff
@@ -17,6 +17,12 @@
     "port_status": 6001
   },
   "ip_address_family" : "IPV6",
+  "socket_options": {
+    "connection_delay_interval": 2,
```

Comment:
Better to add some comments for easy understanding

Reply:
I can't add comments to a JSON file